cypress/integration/orders/orders.openOrderForPhysicalMaterial.spec.js

```diff
@@ -0,0 +1,47 @@
+import TopMenu from '../../support/fragments/topMenu';
+import NewOrder from '../../support/fragments/orders/newOrder';
+import Orders from '../../support/fragments/orders/orders';
+import TestType from '../../support/dictionary/testTypes';
+import OrderLines from '../../support/fragments/orders/orderLines';
+import getRandomPostfix from '../../support/utils/stringTools';
+import InventorySearch from '../../support/fragments/inventory/inventorySearch';
+import newOrganization from '../../support/fragments/organizations/newOrganization';
+
+describe('orders: create an order', () => {
+  const organization = { ...newOrganization.defaultUiOrganizations };
+  const order = { ...NewOrder.defaultOrder };
+  const orderLineTitle = `Autotest Tetle_${getRandomPostfix()}`;
+
+  before(() => {
+    cy.getAdminToken();
+    cy.createOrganizationApi(organization);
+    order.vendor = organization.name;
+    order.orderType = 'One-time';
+    cy.visit(TopMenu.ordersPath);
+  });
+
+  after(() => {
+    cy.getOrdersApi()
+      .then(body => {
+        cy.deleteOrderApi(body.id);
+      });
+    cy.getOrganizationApi({ query: `name="${organization.name}"` })
```

Comment:
teh same, catch organization id and remove it directly without extra organizations getting

Reply:
Done